tests/tests/rpc.rs

```diff
@@ -59,23 +173,38 @@ async fn get_block_range__can_get_serialized_block_from_rpc() {
         .unwrap()
         .payload
     {
-        block
+        remote_info
     } else {
         panic!("expected literal block payload");
     };
-    let ProtoVersionedBlock::V1(v1_block) = actual_block.versioned_block.unwrap();
 
-    let actual_height = match v1_block.header.unwrap().versioned_header.unwrap() {
-        ProtoVersionedHeader::V1(v1_header) => v1_header.height,
-        ProtoVersionedHeader::V2(v2_header) => v2_header.height,
-    };
     // then
-    assert_eq!(expected_header.height.0, actual_height);
+    let key = block_height_to_key(&expected_height);
+    let expected = ProtoRemoteBlockResponse {
+        location: Some(Location::S3(RemoteS3Bucket {
+            bucket: "test-bucket".to_string(),
+            key,
+            requester_pays: false,
+            endpoint: Some("http://127.0.0.1:4566".to_string()),
```

Comment:
You can put string into a constant and reuse in all tests(plus add a comment that it is Localstack address)